### ISSUE TITLE
[Drupal] Create vmsinstall.sh script

### ISSUE THREAD

**Issue by AnthonyAstige** [codeforeauclaire/foodtrucks#2]
**Ensure before closing this issue**
1. ~~#8 is complete~~
1. Data from #7 is pulled in
1. [/drupal-project/README.md](https://github.com/codeforeauclaire/foodtrucks/blob/master/drupal-project/README.md) has functional and smooth vms quickinstall instructions added
1. All TODO notes in /drupal-project/bin/vmsquickinstall.sh are addressed
1. Open a new issue to speed up the scripts execution.  There's a few `sudo apt-get` commands that could be combined.

**Reference**
- Build similar to [documented for the [consumer-ui]](https://github.com/codeforeauclaire/foodtrucks/tree/master/foodtrucks-consumer-ui)
- Use [/drupal-project/README.md](https://github.com/codeforeauclaire/foodtrucks/blob/master/drupal-project/README.md) to figure out what/how to script
- Run during dev with [vms::Setting up other sites vmsquickinstall.sh scripts](https://github.com/codeforeauclaire/vms#setting-up-other-sites-vmsquickinstallsh-scripts)

**Comment by AnthonyAstige** [comment]
I've just now done a fair amount of work on [/drupal-project/bin/vmsquickinstall.sh](https://github.com/codeforeauclaire/foodtrucks/blob/master/drupal-project/bin/vmsquickinstall.sh) in various small commits.  I think a good LAMP stack is running with compatible versions.

Now blocked by #1, perhaps to discuss with @Lowell20 at next meetup in person and continue work on vmsinstall.sh script together in tandem.

**Comment by AnthonyAstige** [comment]
Next step -- setup nginx.

Lowell & I tried https://www.nginx.com/resources/wiki/start/topics/recipes/drupal/, it was not being nice.  Then the server self destructed.  Continue on from notes in vmsscript.

**Comment by AnthonyAstige** [comment]
Changeset https://github.com/codeforeauclaire/foodtrucks/commit/2b17fdfdcb11516f4baa4ab20fd9996083a2df4e makes `drush cim -y` work on debug branch

**Comment by AnthonyAstige** [comment]
@Lowell20 I've gotten [master::vmsquickinstall.sh](https://github.com/codeforeauclaire/foodtrucks/blob/master/drupal-project/bin/vmsquickinstall.sh) with the help of the [debug branch](https://github.com/codeforeauclaire/foodtrucks/tree/debug/drupal-project) closer to functional as shown attached. Is this close to how it's supposed to look? Just missing data? Debug ideas?

![display](https://cloud.githubusercontent.com/assets/236039/14629958/e1374cd8-05cf-11e6-9272-dac8a2f09c40.png)
![source](https://cloud.githubusercontent.com/assets/236039/14629959/e13b88c0-05cf-11e6-93b7-fab6312d2093.png)

P.S. The `$settings['install_profile'] = 'standard';` is from `web/sites/default/settings.php`.  My [bin/vmsquickinstall.settings.php](https://github.com/codeforeauclaire/foodtrucks/blob/master/drupal-project/bin/vmsquickinstall.settings.php) doesn't have it so presumably a drush call or something is putting it there. I've tried moving it up before the `?>` and I get the same display basically as in above screenshots.

![settings php](https://cloud.githubusercontent.com/assets/236039/14630003/3ae4d3c2-05d0-11e6-94aa-48dc6048ec02.png)

**Comment by Lowell20** [comment]
the closing ?> does not belong there. I think it is a drupal peculiarity, do not close php scripts at end of file.
and we removed the Standard from the core.extensions.yml file, probably didn't need to. This ?> was probably why .
I will clean up the config and see how the script runs later this week.

**Comment by AnthonyAstige** [comment]
Makes sense, trailing ?> removed @ https://github.com/codeforeauclaire/foodtrucks/commit/b7232e8abcd9ba28782172553c42274f203822bd.  Looks about the same.  Sounds good having you dig into the config later this week; re-assign this issue to me when you're done.

**Comment by AnthonyAstige** [comment]
Progress at https://github.com/codeforeauclaire/foodtrucks/blob/master/drupal-project/README.md

Need to take away "WIP" text when it's fully functional.

**Comment by Lowell20** [comment]
Streamline install process by incorporating functionality of the scripts in the drupal-project/scripts directory. They setup drupal's sites/default folder and settings files and also the autoload.php file
